### PR TITLE
update client steps to reflect cloud use

### PR DESCRIPTION
**Pull request by alexthomasTG**

### CODE REVIEW

**Comment by a-m-thomas** [comment]
I reworked the GSQL Client documentation into two sections, one for Server and one for Cloud. Even if we collapse the two different client versions into one, I think we should keep two separate links active.

**Comment by elliotmartinTG** [comment]
We now have the client jar hosted at URLs of this form:
https://tigergraph-release-download.s3.us-west-1.amazonaws.com/gsql_client/3.6.1/gsql_client.jar

@arunramasami can you confirm this will be the long term location of client jars? If so, we should add these to this documentation update.

**Comment by arunramasami** [comment]
I don't think this will be a long-term location. We (and Chengbiao) are
working on the location - this will also be updated on the download page.

@Chengbiao Jin ***@***.***>  - can you confirm?

Thanks,
Arun Ramasami
Principal Product Manager, Tigergraph

**Comment by arunramasami** [comment]
It’s already on the download page. Please check and let me know if there is
any feedback.

Thanks,
Chengbiao

**Comment by arunramasami** [comment]
Checkout here. https://dl.tigergraph.com/download.html

**Review by ElliotMtg2011 — CHANGES_REQUESTED** [review]
Need to add info about client being available at dl.tigergraph.com

**Review by ElliotMtg2011 — APPROVED** [review]

File: modules/gsql-shell/pages/using-a-remote-gsql-client.adoc [hunk 1]
```diff
@@ -73,6 +72,46 @@ alias gsql="java -jar <path_to_gsql_client>"
...
+=== Download the GSQL client
+
+link:https://dl.tigergraph.com/download.html[Download the GSQL client for a TigerGraph solution version directly from this page].
+
+=== Obtain SSL certificate
+Run the following command, replacing `<domain>` with the domain of your Cloud solution and replace `<path_to_certificate>` with the local path to place the certificate file.
+
+Find the domain of a Cloud solution by clicking on it in the solution view under "My Solutions."
+
+You need to install link:https://www.openssl.org/[`openssl`] on your client machine to run this command.
```
Comment (lennessyy) [inline modules/gsql-shell/pages/using-a-remote-gsql-client.adoc:88]: Should add this to prerequisites

**Review by lennessyy — APPROVED** [review]

**Pull request merged**